reservation_units/tests/factories.py

```diff
@@ -109,6 +112,27 @@ class Meta:
         model = "reservation_units.ReservationUnitImage"
 
 
+class ReservationUnitPricingFactory(DjangoModelFactory):
+
+    begins = date(2021, 1, 1)
+    pricing_type = PricingType.PAID
+    price_unit = "per_15_mins"
+    lowest_price = 5
+    highest_price = 10
+    tax_percentage = SubFactory(TaxPercentageFactory, value=10.0)
+    status = "active"
+
+    class Meta:
+        model = "reservation_units.ReservationUnitPricing"
+
+    @post_generation
+    def reservation_unit(self, create, reservation_unit, **kwargs):
```

Comment:
No need for this since this `@post_generation` annotated is usually for adding m2m relationships.

Reply:
Removed 👍